Examples/Python/src/Geant4Component.cpp

```diff
@@ -97,6 +98,88 @@ PYBIND11_MODULE(ActsPythonBindingsGeant4, mod) {
     ACTS_PYTHON_STRUCT_END();
   }
 
+  {
+    using Config = SensitiveSurfaceMapper::Config;
+    auto sm =
+        py::class_<SensitiveSurfaceMapper,
+                   std::shared_ptr<SensitiveSurfaceMapper>>(
+            mod, "SensitiveSurfaceMapper")
+            .def(py::init([](const Config& cfg, Acts::Logging::Level level) {
+              return std::make_shared<SensitiveSurfaceMapper>(
+                  cfg, getDefaultLogger("SensitiveSurfaceMapper", level));
+            }));
+
+    auto c = py::class_<SensitiveSurfaceMapper::Config>(sm, "Config")
+                 .def(py::init<>());
+    ACTS_PYTHON_STRUCT_BEGIN(c, SensitiveSurfaceMapper::Config);
+    ACTS_PYTHON_MEMBER(materialMappings);
+    ACTS_PYTHON_MEMBER(volumeMappings);
+    ACTS_PYTHON_MEMBER(candidateSurfaces);
+    ACTS_PYTHON_STRUCT_END();
+
+    sm.def(
+        "create", [](const Config& cfg, Acts::Logging::Level level,
+                     const std::shared_ptr<const TrackingGeometry> tGeometry) {
+          // Helper struct to find the sensitive surface candidates
+          struct SensitiveCandidates {
+            std::shared_ptr<const TrackingGeometry> trackingGeometry;
```

Comment:
Would it make sense to have helpers for that in C++?

Having the code here forces you to rewrite it if you are calling G4 from C++

Reply:
I pulled the TrackingGeometry part back into C++, not the `Experimental::Detector` one. We want to test the experimental detector w/o dragging the `Experimental` code everywhere for the moment.